src/PerformanceTests/Tests/Categories/SqlFixture.cs

```diff
@@ -5,7 +5,7 @@ namespace Categories
     using Tests.Permutations;
     using Variables;
 
-    [TestFixture(Category = "Performance"), Explicit]
+    [TestFixture(Category = "Performance")]
```

Comment:
Again, already covered by PersistenceFixture.

Reply:
PersistenceFixture currently isn't configured for net core.